packages/font-icons/src-cs/Telerik.FontIcons/Icons/FontIcons.cs

```diff
@@ -0,0 +1,652 @@
+// This file is auto-generated by the build process.
```

Comment:
We better use the singular name `FontIcon` instead of `FontIcons` - this is fine for the SvgIcon class, so it's probably a typo.

Reply:
Yes, the `enum` name should be singular, as in the [reference spike project](https://github.com/radkostanev/icons-spike/blob/master/Telerik.FontIcons/Icons/FontIcon.cs#L4).